### PR TITLE
Add LFS control

### PR DESCRIPTION
**Pull request by dsm-kb**
Add ability to turn LFS on / off for repo

### CODE REVIEW

File: github/repos_lfs.go
Comment (gmlewis) [inline github/repos_lfs.go:1]: For new files, we give them the year that they are created.

```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```

File: github/repos_lfs_test.go
Comment (gmlewis) [inline github/repos_lfs_test.go:1]:
```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```

File: github/repos_lfs.go
Comment (gmlewis) [inline github/repos_lfs.go:13]:
```suggestion
// EnableLFS turns the LFS (Large File Storage) feature ON for the selected repo.
```

File: github/repos_lfs.go
Comment (gmlewis) [inline github/repos_lfs.go:16]:
```suggestion
func (s *RepositoriesService) EnableLFS(ctx context.Context, owner, repo string) (*Response, error) {
```

File: github/repos_lfs.go
Comment (gmlewis) [inline github/repos_lfs.go:32]:
```suggestion
// DisableLFS turns the LFS (Large File Storage) feature OFF for the selected repo.
```

File: github/repos_lfs.go
Comment (gmlewis) [inline github/repos_lfs.go:35]:
```suggestion
func (s *RepositoriesService) DisableLFS(ctx context.Context, owner, repo string) (*Response, error) {
```

File: github/repos_lfs_test.go
Comment (gmlewis) [inline github/repos_lfs_test.go:29]:
```suggestion
	const methodName = "EnableLFS"
```

File: github/repos_lfs_test.go
Comment (gmlewis) [inline github/repos_lfs_test.go:55]:
```suggestion
	const methodName = "DisableLFS"
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @dsm-kb !
Just a few minor tweaks, please, then we should be ready for a second LGTM before merging.

File: github/repos_lfs.go
Comment (gmlewis) [inline github/repos_lfs.go:32]:
```suggestion
// DisableLFS turns the LFS (Large File Storage) feature OFF for the selected repo.
```

File: github/repos_lfs.go
Comment (gmlewis) [inline github/repos_lfs.go:34]:
```suggestion
// GitHub API docs: https://docs.github.com/en/rest/repos/lfs#disable-git-lfs-for-a-repository
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Sorry I missed a couple minor nits.

**Review by gmlewis — APPROVED** [review]
Thank you, @dsm-kb !
LGTM.

Awaiting second LGTM from any other contributor to this repo before merging.

**Comment by dsm-kb** [comment]
Thank you @gmlewis for the LGTM :-)

**Review by raynigon — APPROVED** [review]
LGTM

@dsm-kb For ypur next PR it would be nice, if you could explain what you want to do in the PR description. Makes the review a bit easier :)

**Comment by gmlewis** [comment]
Thank you, @raynigon !
Merged.

**Pull request merged**
